buildrpm/python-drgn-tools.spec

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2024, Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
 Name:           python-drgn-tools
-Version:        0.9.1
+Version:        1.0.0
```

Comment:
Here is the main branch, so it is still 1.0.0? You will create a new branch for v1 and set version to 1.1.0 there and also change the version here to 2.0.0, right?

Reply:
No, I totally missed that. This should go to 1.1.0, I'll create the `stable/v1` branch, and then on the main branch I'll bump to 2.0.0.